pyvista/plotting/plotting.py

```diff
@@ -968,16 +968,106 @@ def enable_lightkit(self, only_active=False):
+            Anti-aliasing type. See the notes below. One of the following:
+
+            * ``"ssaa"`` - Super-Sample Anti-Aliasing
+            * ``"mxaa"`` - Multi-Sample Anti-Aliasing
+            * ``"fxaa"`` - Fast Approximate Anti-Aliasing
+
+        multi_samples : int, optional
+            The number of multi-samples when ``aa_type`` is ``"mxaa"``. Note
+            that using this setting automatically enables this for all
+            renderers.
+
+        all_renderers : bool
+            If ``True``, applies to all renderers in subplots. If
+            ``False``, then only applies to the active renderer.
+
+        Notes
+        -----
+        SSAA, or Super-Sample Anti-Aliasing is a brute force method of
+        anti-aliasing. It results in the best image quality but comes at a
+        tremendous resource cost. SSAA works by rendering the scene at a higher
+        resolution. The final image is produced by downsampling the
+        massive source image using an averaging filter. This acts as a low pass
+        filter which removes the high frequency components that caused the
+        jaggedness.
+
+        MSAA, or Multi-Sample Anti-Aliasing is an optimization of SSAA that
+        reduces the amount of pixel shader evaluations that need to be computed
+        by focusing on overlapping regions of the scene. The result is
+        antialiasing along edges that is on par with SSAA and less
+        anti-aliasing along surfaces as these make up the bulk of SSAA
+        computations. MSAA is substantially less computationally expensive than
+        SSAA and results in comparable image quality.
+
+        FXAA, or Fast Approximate Anti-Aliasing is an Anti-Aliasing technique
+        that is performed entirely in post processing. FXAA operates on the
+        rasterized image rather than the scene geometry. As a consequence,
+        forcing FXAA or using FXAA incorrectly can result in the FXAA filter
+        smoothing out parts of the visual overlay that are usually kept sharp
+        for reasons of clarity as well as smoothing out textures. FXAA is
+        inferior to MSAA but is almost free computationally and is thus
+        desirable on low end platforms.
 
-    @wraps(Renderer.disable_anti_aliasing)
-    def disable_anti_aliasing(self, *args, **kwargs):
-        """Wrap ``Renderer.disable_anti_aliasing``."""
-        self.renderer.disable_anti_aliasing(*args, **kwargs)
+        Examples
+        --------
+        Enable super-sample anti-aliasing (SSAA)
+
+        >>> import pyvista
+        >>> pl = pyvista.Plotter()
+        >>> pl.enable_anti_aliasing('ssaa')
+        >>> _ = pl.add_mesh(pyvista.Sphere(), show_edges=True)
+        >>> pl.show()
+
+        """
+        # apply MXAA to entire render window
+        if aa_type == 'mxaa':
+            if hasattr(self, 'ren_win'):
+                self.ren_win.SetMultiSamples(multi_samples)
+            else:
+                self.theme.multi_samples = multi_samples
```

Comment:
Half of this `if` is not covered according to codecov, and I don't understand the mechanics here to be honest. Is `SetMultiSamples(multi_samples)` exactly MSAA? And if there's no render window (which seems to always be the case according to code cov), just set this for the theme? Why set it for the theme, and why not set it for the first conditional branch?

Reply:
Fix was just to move the set `enable_anti_aliasing` to the end of the Plotter initialization in 69243a9ff453f7d292ce9329026b0ef8c43fa355.